testsupport/tiers/templaterefs.go

```diff
@@ -32,3 +37,33 @@ func clusterResourcesRevision(tier toolchainv1alpha1.NSTemplateTier) *string {
 	}
 	return nil
 }
+
+// ComputeTemplateRefsHash computes the hash of the `.spec.namespaces[].templateRef` + `.spec.clusteResource.TemplateRef`
+func ComputeTemplateRefsHash(tier *toolchainv1alpha1.NSTemplateTier) (string, error) {
+	refs := []string{}
+	for _, ns := range tier.Spec.Namespaces {
+		refs = append(refs, ns.TemplateRef)
+	}
+	if tier.Spec.ClusterResources != nil {
+		refs = append(refs, tier.Spec.ClusterResources.TemplateRef)
+	}
+	sort.Strings(refs)
+	m, err := json.Marshal(templateRefs{Refs: refs})
+	if err != nil {
+		return "", err
+	}
+	md5hash := md5.New()
+	// Ignore the error, as this implementation cannot return one
+	_, _ = md5hash.Write(m)
+	hash := hex.EncodeToString(md5hash.Sum(nil))
+	return hash, nil
+}
```

Comment:
Looks like we have the same function here: https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/test/masteruserrecord/masteruserrecord.go#L125

Maybe we should just move that one to another package in toolchain-common and reference it?

Reply:
Actually, maybe this whole file can be moved to toolchain-common

Reply:
good point - I wasn't aware of that. Move in https://github.com/codeready-toolchain/toolchain-common/pull/230

and I moved only these new functions - the other ones that are in this file are e2e specific, so I would keep them here